db/models.py

```diff
@@ -1 +1,49 @@
 from django.db import models
+
+
+class Race(models.Model):
+    NAMES_CHOICES = [
+        ("ELF", "Elf"),
```

Comment:
the first element in tuple here should match exactly the name that you would expect from your file ('elf', 'dwarf' etc) and that will be stored in db, second can be a human readable name which you will display to user